ipatests/test_webui/test_pwpolicy.py

```diff
@@ -54,4 +51,214 @@ def test_crud(self):
+        self.wait_for_request(d=0.5)
+        self.assert_notification(assert_text='1 item(s) deleted')
+        self.assert_record(pwpolicy.PKEY3, negative=True)
+        self.close_notifications()
+
+        # test delete multiple records
+        records = [pwpolicy.DATA, pwpolicy.DATA1, pwpolicy.DATA2]
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        self.select_multiple_records(records)
+        self.facet_button_click('remove')
+        self.dialog_button_click('ok')
+
+    @screenshot
+    def test_negative_value(self):
+        """
+        Negative test for Password policy fields in edit page
+        """
+        self.init_app()
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        non_interger_expected_error = 'Must be an integer'
+        minimum_value_expected_error = 'Minimum value is 0'
+        non_integer = 'nonInteger'
+        maximum_value = '2147483649'
+        minimum_value = '-1'
+
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA1,
+                        dialog_btn='add_and_edit')
+
+        for field in FIELDS:
+            # bigger than max value
+            if field == 'krbmaxpwdlife':
+                self.modify_policy(field, 'Maximum value is 20000',
+                                   maximum_value)
+            elif field == 'krbpwdmindiffchars':
+                self.modify_policy(field, 'Maximum value is 5', maximum_value)
+            else:
+                self.modify_policy(field, 'Maximum value is 2147483647',
+                                   maximum_value)
+
+            # string used instead of integer
+            self.modify_policy(field, non_interger_expected_error,
+                               non_integer)
+
+            # smaller than max value
+            self.modify_policy(field, minimum_value_expected_error,
+                               minimum_value)
+        self.navigate_to_entity(pwpolicy.ENTITY)
+        self.delete_record(pwpolicy.group.PKEY)
+
+        # Negative test for policy priority
+        field_priority = 'cospriority'
+        self.add_record(pwpolicy.ENTITY, pwpolicy.DATA7, dialog_btn=None)
+        # non integer for policy priority
+        self.fill_input(field_priority, non_integer)
+        self.assert_field_validation(non_interger_expected_error,
+                                     field=field_priority)
+        #  lower bound of data range
+        self.fill_input(field_priority, minimum_value)
+        self.assert_field_validation(minimum_value_expected_error,
+                                     field=field_priority)
+        # upper bound of data range
+        self.fill_input(field_priority, maximum_value)
+        self.assert_field_validation(expect_error='Maximum value is'
+                                                  ' 2147483647',
+                                     field=field_priority)
+        self.close_all_dialogs()
+
+    def modify_policy(self, pwdfield, expected_error, value):
```

Comment:
the method name doesn't describe the purpose much. The intention here is not to modify the policy (the actual mod is not even executed) but rather test field validation after changing values. The method name should reflect that.